cirq/optimizers/eject_z.py

```diff
@@ -26,6 +27,18 @@
     from typing import Dict, List, Tuple
 
 
+def _is_swaplike(op: ops.Operation):
+    gate1 = ops.op_gate_of_type(op, ops.SwapPowGate)
+    if gate1:
+        return gate1.exponent == 1
+
+    gate2 = ops.op_gate_of_type(op, ops.FSimGate)
+    if gate2:
+        return np.isclose(gate2.theta, np.pi / 2)
```

Comment:
should this also include other angles that give swap like behavior?

Reply:
Yeah, `-np.pi / 2` also gives swap-like behavior.

Reply:
done